pkg/apis/garden/helper/helpers_test.go

```diff
@@ -297,4 +297,26 @@ var _ = Describe("helper", func() {
 		Entry("secret", "v1", "Secret", "secret", BeNil()),
 		Entry("unknown", "v2", "Foo", "", HaveOccurred()),
 	)
+
+	var (
+		unmanagedType = "unmanaged"
```

Comment:
```suggestion
		unmanagedType = garden.DNSUnmanaged
```
Or simply use `garden.DNSUnmanaged` below instead of `unmanagedType`